ui/00_message/common/item_info.xml

```diff
@@ -9,8 +9,7 @@
   </item>
   <item id="36">
     <original>使用者のスタミナを回復する</original>
-    <translation>Cures the user of the specified
-status debilitations.</translation>
+    <translation>Recovers a portion of the user´s Stamina.</translation>
```

Comment:
Interesting that this error was made in the original translation.

You shouldn't use "´". Use a normal "'".